control_panel_api/settings/test.py

```diff
@@ -8,3 +8,5 @@
 LOGS_BUCKET_NAME = 'moj-test-logs'
 IAM_ARN_BASE = 'arn:aws:iam::123'
 K8S_WORKER_ROLE_NAME = 'test-k8s-worker-role'
+SAML_PROVIDER_ARN = 'arn:aws:iam::12:saml-provider/test'
+K8S_WORKER_ROLE_ARN = 'arn:aws:iam::123:role/test'
```

Comment:
As said in other comment, this setting is not necessary, we're generating it by using `IAM_ARN_BASE` and `K8S_WORKER_ROLE_NAME` (this will probably be clear when you'll refactor `create_app_role()` and `create_user_role()`.